FGECore/EntitySystem/JointSystems/PhysicsJoints/Jointangularservo.cs

```diff
@@ -0,0 +1,61 @@
+﻿//
+// This file is part of the Frenetic Game Engine, created by Frenetic LLC.
+// This code is Copyright (C) Frenetic LLC under the terms of a strict license.
+// See README.md or LICENSE.txt in the FreneticGameEngine source root for the contents of the license.
+// If neither of these are available, assume that neither you nor anyone other than the copyright holder
+// hold any right or permission to use this software until such time as the official license is identified.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using FGECore.MathHelpers;
+using BepuPhysics.Constraints;
+
+namespace FGECore.EntitySystem.JointSystems.PhysicsJoints;
+
+/// <summary>Drives rotation of a local axis toward a target angle using a servo, with spring/damping and speed/force limits.
+/// The target can be changed at runtime via <see cref="SetTargetAngle"/>.</summary>
+public class JointAngularServo(EntityPhysicsProperty e1, EntityPhysicsProperty e2, Location axis) : PhysicsJointBase<AngularServo>(e1, e2)
+{
+    /// <summary>Local axis on <see cref="PhysicsJointBase.One"/> to rotate around.</summary>
+    public Location Axis = axis;
+
+    /// <summary>Target angle (radians) of the <see cref="Axis"/> of <see cref="PhysicsJointBase.One"/>.</summary>
+    public float TargetAngle = 0;
+
+    /// <summary>Maximum speed this servo can try to move at.</summary>
+    public float MaximumSpeed = float.MaxValue;
+
+    /// <summary>Maximum amount of force this servo can output.</summary>
+    public float MaximumForce = float.MaxValue;
```

Comment:
Similar field names should be direct copied from `JointAxisServo` as a reference as well